dts/bindings/led_strip/ti,tlc59731.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Javad Rahimipetroudi <javad.rahimipetroudi@mind.be>
+# SPDX-License-Identifier: Apache-2.0
+
+compatible: "ti,tlc59731"
+description: TLC59731 RGB LED Controller
+
+properties:
+  gpios:
+    required: true
+    type: phandle-array
+    description: |
+      GPIO to send command data to the controller.
+  label:
```

Comment:
I don't think led strips typically have a label. Also, this label property isn't handled in the driver code.